frontend/src/hooks/useLayoutManager.tsx

```diff
@@ -4,7 +4,8 @@ import LayoutManager from '../utils/layoutManager';
 
 export type GetLayout = (
   containerDimensions: { height: number; width: number },
-  boxes: Element[]
+  boxes: Element[],
+  fixBigRatio: boolean
```

Comment:
Nit: the name is leaking an implementation detail.  Reading through the code, it looks like a configuration for prioritizing landscape mode over portrait mode.

Maybe this can be renamed to reflect that somehow?